WordPress/src/main/java/org/wordpress/android/ui/posts/adapters/PostsListAdapter.java

```diff
@@ -309,6 +305,46 @@ public void onClick(View v) {
         });
     }
 
+    private void showFeaturedImage(int postId, WPNetworkImageView imgFeatured) {
+        String imageUrl = mFeaturedImageUrls.get(postId);
+        if (imageUrl == null) {
+            imgFeatured.setVisibility(View.GONE);
+        } else if (imageUrl.startsWith("http")) {
+            String photonUrl =  ReaderUtils.getResizedImageUrl(imageUrl, mPhotonWidth, mPhotonHeight, mSite.isPrivate());
+            imgFeatured.setVisibility(View.VISIBLE);
+            imgFeatured.setImageUrl(photonUrl, WPNetworkImageView.ImageType.PHOTO);
+        } else {
+            Bitmap bmp = fastResizeBitmap(imageUrl, mPhotonWidth);
```

Comment:
Any reason to not use `Bitmap bmp = ImageUtils.getWPImageSpanThumbnailFromFilePath(context, imageUrl, mPhotonWidth);`  that is well tested and used in the app?

The actual implementation doesn't take in consideration orientation of the pictures, resulting in pictures turned 90° on some devices, and doesn't show the "poster" picture for videos.